File: JavaLox/src/com/craftinginterpreters/lox/LinkedList.java
package com.craftinginterpreters.lox;

import java.util.List;

class LinkedList implements LoxCallable {
   @Override
   public int arity() {
      return 0;
   }

   @Override
   public Object call(Interpreter interpreter, List<Object> arguments) {
      return new LoxLinkedList();
   }
}

File: CsLox/com/craftinginterpreters/lox/LinkedList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace com.craftinginterpreters.lox
{
    internal class LinkedList : LoxCallable
    {
        public virtual int arity()
        {
            return 0;
        }

        public virtual object call(Interpreter interpreter, List<object> arguments)
        {
            return new LoxLinkedList();
        }
    }
}
